Makefile

```diff
@@ -22,6 +22,11 @@ deflake: ## Run randomized, racing tests until the test fails to catch flakes
 		-v \
 		./pkg/...
 
+install-kwok: ## Install kwok into cluster
+	UNINSTALL_KWOK=false ./hack/install-kwok.sh
```

Comment:
optional: I wonder if we should have this be a CLI flag as opposed to an environment variable.

Reply:
Can go either way, what's your thought on the benefits of the CLI flag?

Reply:
CLI flags are a bit easier to reason about when it comes to parameters being passed to scripts. Without a CLI arg here, if I were coming at the script without the Makefile, it would be harder for me to understand what the script is capable of doing and what "special environment variable" I need to set to get Karpenter to uninstall with this script. Not blocking, though. Just a thought.

Reply:
That's why I wanted to use the environment variables and have the two makefile targets of `install-kwok` and `uninstall-kwok`. This adds the environment variables for them, and the script will fail if this is unset anyway.

Reply:
Sounds fine. Only a thought, definitely not required.